### ISSUE TITLE
Text within panel component can overflow the container

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#2173]
## Description of the issue

As originally raised in https://github.com/hmrc/accessibility/issues/20, and flagged in the backlog [here](https://github.com/alphagov/govuk-design-system-backlog/issues/40#issuecomment-729058084), text within the panel component can overflow the container, for example if viewed on a mobile with text resized to 200%.

![](https://user-images.githubusercontent.com/272769/82329510-53bbf000-99d9-11ea-8c4a-128de706ac17.png)

## Steps to reproduce the issue

In Safari on iOS

1. Open http://govuk-frontend-review.herokuapp.com/components/panel/preview on a mobile device
2. Resize the text to 200%

## Actual vs expected behaviour

The text should remain readable, but the words 'Application' and 'completed' are truncated (because they are white text on a white background where they overflow the container).

**Comment by vanitabarrett** [comment]
This came up on support today, again flagged by an internal accessibility audit at HMRC as failing WCAG 1.4.4.

> Issue: Text is clipped (AA)
> WCAG 2.1 violation: 1.4.4 Resize text
> Page: Application Submitted
> URL:
> Journey/Step: J1 S6
> Description: A long string of text not breaking to accommodate the width of the parent container. The text gets clipped. Tested with increased page zoom 200% on ios using iPhone11
> Resolve: Text getting clipped due to text doesn’t wrap. Apply appropriate CSS.

**Comment by vanitabarrett** [comment]
Had a quick look at this, using the solution mentioned above:

```scss
    word-wrap: break-word; // Fallback for older browsers only
    overflow-wrap: break-word;
```

It's definitely an improvement, as the text no longer overflows (white on white) so you can read it. On very small width screens it can be difficult to read because the text is so broken up.

<img width="144" alt="Screenshot 2021-09-09 at 16 16 43" src="https://user-images.githubusercontent.com/29889908/132713751-b3ac4724-25cf-462d-b98a-9a798ec50a76.png">

The other alternative would probably be combining the above solution with something like `min-content` for newer browsers, e.g:

```scss
    word-wrap: break-word; // Fallback for older browsers only
    overflow-wrap: break-word;

    @include govuk-media-query($until: tablet) {
      padding: govuk-spacing(6) - $govuk-border-width;
    }

    @supports (width: min-content) {
      min-width: min-content;
    }
```

Text is more readable, but it means the panel overflows the page which looks a bit broken and you have to scroll to see all the content.

<img width="153" alt="Screenshot 2021-09-09 at 16 18 21" src="https://user-images.githubusercontent.com/29889908/132714095-7b211c89-6dba-42cf-8640-46d16234f4fc.png">

**Comment by paulwaitehomeoffice** [comment]
Yeah it's not ideal eh; ours can be really tricky to read when zoomed in more:

![Default font size on our confirmation page on a 320 viewport](https://user-images.githubusercontent.com/25032977/132716808-636581e7-0bdb-401f-9930-90d7066def79.png)

![Zoomed font size on our confirmation page on a 320 viewport](https://user-images.githubusercontent.com/25032977/132716830-079289f5-e15d-4c68-ab19-58d61e9d49fd.png)

Not sure if there's any scope for reducing the panel's padding on smaller viewports, although obviously that would only help a bit.

Marat did mention some other word wrapping-related CSS that might be worth trying, even if not supported everywhere yet, that might be worth trying too:

```
  word-wrap: break-word; /* old version of overflow-wrap for IE and Edge and FF < 49*/
  overflow-wrap: anywhere; /* modern, tries to fit word on a new line rather than fitting on remnants of the current line */
  /* hyphens is not supported on Chrome < 55 or Opera and misc mobile browsers */
  -webkit-hyphens: auto;
  -ms-hyphens: auto; /* ie10+ */
  -moz-hyphens: auto;
  hyphens: auto;
```
